galaxy_ng/app/migrations/0003_inbound_repo_per_namespace.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+def create_inbound_repo_per_namespace(apps, schema_editor):
+    AnsibleRepository = apps.get_model('ansible', 'AnsibleRepository')
+    AnsibleDistribution = apps.get_model('ansible', 'AnsibleDistribution')
+    Namespace = apps.get_model('galaxy', 'Namespace')
+
+    for namespace in Namespace.objects.all():
```

Comment:
Better specify db_alias explicitly as it's recommended in Django [docs](https://docs.djangoproject.com/en/3.0/ref/migration-operations/#runpython).

Reply:
Nice tip!

Reply:
Added db_alias